public/css/components.css

```diff
@@ -240,6 +242,7 @@ a.button {
   text-decoration: none;
   padding-bottom: 5px;
   color: #333;
+  white-space: nowrap;
```

Comment:
This is a change that you need to make site-wide, so it _must_ be here, in the component. However, to modify a component, I'd create a separate PR, and also you'll need to update the Pattern Library and several other things. See the TeamCode website for more info. This is part of an "atom".

Reply:
This change needs to be completely removed from this PR and put into a separate PR.

Reply:
I'm confused because you already made this change here https://github.com/team-refcode/refcode.org/pull/291/files#diff-9325f16d0687001c20b5a63e815d42042a4db132ab12bf2e827ae5f2ef26f7e7R243 so how is GitHub seeing this as an addition?